kentik_api_library/README.md

```diff
@@ -16,6 +16,45 @@ For more information on how to interact with kentik resources using this library
   python3 examples/sites_example.py
   ```
 
+## Example - create a new device
+
+```python
+from kentik_api import (
+    KentikAPI,
+    Device,
+    DeviceSubtype,
+    CDNAttribute,
+    ID,
+)
+
+# initialize Kentik API client
+api_client = KentikAPI("<API_EMAIL_STRING>", "<API_TOKEN_STRING>")
+
+# prepare device object
+device = Device.new_dns(
+    device_name="example-device-1",
+    device_subtype=DeviceSubtype.aws_subnet,
+    cdn_attr=CDNAttribute.yes,
+    device_sample_rate=100,
+    plan_id=ID(11466),
+    site_id=ID(8483),
+    device_bgp_flowspec=True,
+)
+
+# create the device
+created = api_client.devices.create(device)
+
+# print returned device's attributes
+print(created.__dict__)
+```
+
+## Where to find options related to given resource type
+
+The general approach is that all the types/enums/constants related to a given resource type are collected together with the resource class (in the same source file), e.g. all types describing:
+- Device - are located in kentik_api_library/kentik_api/public/device.py
```

Comment:
Improvement: make it a link `[]()` so that user can click to jump to the file.

Reply:
I was thinking about this but then the obvious question is why include links to just some resources not all of them, not sure if we want to go that way?

Reply:
Finally, reworked the entire example paragraph